### DISCUSSION TITLE
How to make TextField square.

### DISCUSSION THREAD

**Discussion by YukiWorks432** [adobe/react-spectrum#7333] [Q&A]
The TextField seems to ignore the `height`, `minHeight` and `maxHeight` settings.
Is there any way to resolve this and make the TextField look square?
There is also a problem with `width`, when the value is made smaller and smaller, the appearance does not change from a certain size, even if `minWidth` and `width` are set to the same value.

**Comment by reidbarber** [comment]
Is this for React Spectrum? The styles such as height and minWidth are set by design, so they won't be customizable. Does [TextArea](https://react-spectrum.adobe.com/react-spectrum/TextArea.html) fit your design more?

If this is for React Aria, you should have total control over the styling: https://react-spectrum.adobe.com/react-aria/TextField.html#styling
